lib/Quantization/Quantization.cpp

```diff
@@ -420,29 +420,32 @@ class FunctionQuantizer : public FunctionConverter {
       // [         RowwiseQuantizedFullyConnectedNode            ]
       //                              |
       //                       [DequantizeNode]
-      if (auto *Q = llvm::dyn_cast<DequantizeNode>(&node)) {
-        if (auto *fcN = llvm::dyn_cast<FullyConnectedNode>(Q->getInput())) {
-          NodeValue input = fcN->getInput();
-          NodeValue weights = fcN->getWeights();
-          NodeValue bias = fcN->getBias();
-          NodeValue result = fcN->getResult();
-          // Only convert quantized FullyConnected Node.
-          if (input.getType()->isQuantizedType() &&
-              llvm::isa<QuantizeNode>(weights.getNode()) &&
-              bias.getType()->isQuantizedType() &&
-              result.getType()->isQuantizedType()) {
-            auto *wq = llvm::dyn_cast<QuantizeNode>(weights.getNode());
-            // For RowwiseQuantizedFullyConnected, the weights need to be
-            // constant.
-            if (Constant *wc = llvm::dyn_cast<Constant>(wq->getInput())) {
-              auto *fcq = function_.createRowwiseQuantizedFullyConnected(
-                  "rowwiseqfc", input, wc, bias, result.getType(),
-                  /* transposeWeight */ true);
-              // Replace the usages of quantized FC node to
-              // RowwiseQuantizedFullyConnected Node.
-              result.replaceAllUsesOfWith(fcq->getResult());
-            }
-          }
+      auto *Q = llvm::dyn_cast<DequantizeNode>(&node);
+      if (!Q) {
+        continue;
+      }
+      auto *fcN = llvm::dyn_cast<FullyConnectedNode>(Q->getInput());
+      if (!fcN) {
+        continue;
+      }
+      NodeValue input = fcN->getInput();
+      NodeValue weights = fcN->getWeights();
+      NodeValue bias = fcN->getBias();
+      NodeValue result = fcN->getResult();
+      // Only convert quantized FullyConnected Node.
+      if (input.getType()->isQuantizedType() &&
+          llvm::isa<QuantizeNode>(weights.getNode()) &&
+          bias.getType()->isQuantizedType() &&
+          result.getType()->isQuantizedType()) {
+        auto *wq = llvm::dyn_cast<QuantizeNode>(weights.getNode());
+        // For RowwiseQuantizedFullyConnected, the weights need to be constant.
+        if (Constant *wc = llvm::dyn_cast<Constant>(wq->getInput())) {
+          auto *fcq = function_.createRowwiseQuantizedFullyConnected(
+              "rowwiseqfc", input, wc, bias, result.getType(),
+              /* transposeWeight */ true);
+          // Replace the usages of quantized FC node to
+          // RowwiseQuantizedFullyConnected Node.
+          result.replaceAllUsesOfWith(fcq->getResult());
```

Comment:
For the above diff,  it seems to me that only the code structure has been changed: 
`if (A) { if (B) {...} } => A =..; if (!A) continue; B=..; if (!B) continue; ...` ?  If so, I don't think it is related to this PR. Please let me know if I missed anything.

Reply:
Ok, I've removed it.